nyc_taxis/TEST_PROCEDURES.md

```diff
@@ -0,0 +1,55 @@
+# Test Procedures
+
+## searchable_snapshot 
+
+A test procedure for measuring performance for Searchable Snapshots feature. It runs the same search query as the default test procedure `append-no-conflicts`.
+
+In contrast with `append-no-conflicts` which running query on index with local storage, this test procedure runs query on index backed by a remote snapshot.
+
+The test procedure will create a remote snapshot that stored in Amazon S3, so an Amazon S3 bucket for storing the snapshot and an AWS account user credential that has permission to access the bucket are required to run the test procedure. To learn more about configuring Amazon S3 as a snapshot repository, see the [OpenSearch docs](https://opensearch.org/docs/latest/opensearch/snapshots/snapshot-restore#amazon-s3).
+
+Searchable Snapshots feature is supported by OpenSearch since version 2.4.0, see the [OpenSearch docs](https://opensearch.org/docs/2.4/opensearch/snapshots/searchable_snapshot) to learn more.
+
+### Parameters
+
+#### The test procedure allows to overwrite the following parameters using `--workload-params`:
+
+Additional parameters other than those mentioned in [README](README.md) of NYC taxis workload:
+* `snapshot_repository_name` (default: "test-repository"): Name of the snapshot repository.
+* `snapshot_name` (default: "test-snapshot"): Name of the snapshot. 
+  It's recommended to assign a different value for different test executions, because there is no operation defined by opensearch-benchmark to delete a snapshot, and new snapshot won't be created if a snapshot with the same name exists in the repository.
```

Comment:
It is recommended to ...
and a new snapshot will not be created if a previous snapshot with ...

Also, have you considered adding a custom operation to delete the snapshot [on the lines of this example?](https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/main/http_logs/workload.py#L8-L16)

Reply:
@gkamat Thanks for pointing the code to me about adding a custom operation, I didn't know that, and will take a look at.

Reply:
Added the custom operation in a new commit https://github.com/opensearch-project/opensearch-benchmark-workloads/pull/58/commits/f62e62883d8d365fa621be6d87b835eaf53a06da. 😄

Reply:
Looks good.   Please submit a PR for this when ready.

Reply:
Thank you! But no more PR is needed, the code change of the commit https://github.com/opensearch-project/opensearch-benchmark-workloads/pull/58/commits/f62e62883d8d365fa621be6d87b835eaf53a06da has already been in this PR.